sites/geohub/src/components/Settings.svelte

```diff
@@ -0,0 +1,169 @@
+  import { footerItems } from '$lib/constants'
+  import { createHeaderLinks } from '$lib/helper'
+
+  let isExpanded = true
+  let headerHeight: number
+  let activeSettingTab = 'Geohub'
+  let links: HeaderLink[] = createHeaderLinks(['home', 'dashboard', 'userguide'])
+  let applySettingsLoading = false
+
+  // Default Settings
+  const settings = {
+    sideBarPosition: 'left',
+  }
+
+  const getCookie = (name: string) => {
+    const value = `; ${document.cookie}`
+    const parts = value.split(`; ${name}=`)
+    if (parts.length === 2) return parts.pop().split(';').shift()
+  }
+
+  let settingsFromCookies = JSON.parse(getCookie('settings') || '{}')
+  if (settingsFromCookies) {
+    settings.sideBarPosition = settingsFromCookies.sideBarPosition
+  }
+  let sideBarPosition: 'left' | 'right' = JSON.parse(getCookie('settings') || '{}').sideBarPosition || 'left'
+
+  const setSettingsToCookies = () => {
+    applySettingsLoading = true
+    sessionStorage.setItem('settings', JSON.stringify(settings))
+    document.cookie = `settings=${JSON.stringify(settings)}; path=/;`
+    applySettingsLoading = false
+  }
+
+  const collapseMiniMenu = () => {
+    isExpanded = !isExpanded
+  }
+</script>
+
+<Header
+  region="UNDP's one stop shop for spatial data and analytics"
+  siteTitle="GeoHub"
+  url="https://geohub.data.undp.org"
+  logoUrl="assets/undp-images/undp-logo-blue.svg"
+  bind:height={headerHeight}
+  bind:links
+  isPositionFixed={true}>
+  <div slot="custom-button">
+    <UserAccount />
+  </div>
+</Header>
+
+<div
+  class="columns is-one-quarter ml-auto mr-auto"
+  style="margin-top: {headerHeight}px; z-index:-1">
+  <div class="column is-2">
+    <aside class="menu">
+      <ul class="menu-list">
+        <li>
+          <a
+            on:click={collapseMiniMenu}
+            href="#">GeoHub Settings</a>
+          <ul style="display: {!isExpanded ? 'none' : ''}">
+            <li>
+              <a
+                class={activeSettingTab === 'Geohub' ? 'selected' : ''}
+                on:click={() => (activeSettingTab = 'Geohub')}
+                href="#">Map Settings</a>
+            </li>
+          </ul>
+        </li>
+      </ul>
+    </aside>
+  </div>
+  <div class="column is-three-quarters m-auto">
+    <section
+      class="content {activeSettingTab !== 'Geohub' ? 'is-hidden' : ''}"
+      id="Geohub">
+      <h2>Sidebar Position</h2>
+      <p>Change the sidebar position between left and right</p>
+      <div class="columns">
+        <div
+          on:click={() => (settings.sideBarPosition = 'left')}
+          class="column sidebar-col {settings.sideBarPosition === 'left' ? 'selected-sidebar' : ''}">
+          <div class="card">
+            <div class="card-header">
+              <p class="card-header-title">Left Sidebar</p>
+            </div>
+            <div class="card-content">
+              <div class="content">
+                <img
+                  src="/assets/sidebar/left-sidebar.png"
```

Comment:
I am wondering whether we can use this maximise icon for sidebar position.

https://fontawesome.com/icons/window-maximize?s=regular&f=classic

This screenshots also look good. but I feel the images are too large. I think smaller image is much better. maybe one-third of current image size.